source/fundamentals/data-formats/document-data-format-json.txt

```diff
@@ -0,0 +1,45 @@
+==========================
+Document Data Format: JSON
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+- Describe Extended JSON
+- Question here: what is the difference between toJson() and parse()?
+
+Read JSON
+---------
+
+- Description; mention BsonBinaryReader and JsonReader implementations
+- How to use parse/toJson to output for a BufferedReader
+- Example
+- Link to data-formats/document-data-formats-bson for BsonReaders
+
+Write JSON
+----------
+
+- Description of JsonWriter; mention BsonBinaryWriter and JsonWriter implementations
```

Comment:
BsonBinaryWriter isn't used to write JSON, only JsonWriter

Reply:
Thanks, updated.